ui/components/UserPreferences/index.js

```diff
@@ -457,7 +458,7 @@ const UserPreference = (props) => {
                         <Typography
                           variant="body1"
                           className={props.classes.hideScrollbar}
-                          style={{ marginRight: '20px' }}
+                          style={{ marginRight: '20px', wordBreak: 'break-word' }}
```

Comment:
Thanks, @ChenemiKen!

Generally, we prefer not to break words. Word wrap - ok.

Will you take a screenshot of before and after these changes, @ChenemiKen?

Reply:
noted